state-chain/pallets/cf-broadcast/src/tests.rs

```diff
@@ -200,68 +189,43 @@ fn test_abort_after_number_of_attempts_is_equal_to_the_number_of_authorities() {
 	});
 }
 
+// Helper function: make a broadcast to be aborted upon the next failure.
+fn ready_to_abort_broadcast(broadcast_id: BroadcastId) {
```

Comment:
this would be slightly better if it returned the nominee right? that way we can use the nominee who has not yet failed to submit the transaction, and guarantee it in the test

Reply:
great idea!